server/utilities.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Contains miscellaneous functions and classes.
+ */
+
+
+let Coordinate = {x: a , y: b};
+

```

Comment:
This isn't needed. Instead coordinates can be defined inline where they are used